src/framework/uicomponents/qml/MuseScore/UiComponents/StyledMenu.qml

```diff
@@ -8,6 +8,11 @@ import MuseScore.Ui 1.0
 StyledPopupView {
     id: root
 
+    property alias model: view.model
```

Comment:
I'd advise not to name this alias as a "model", because it can overlap with context variables, which would lead to confusion or issues

Reply:
I just move properties (did reordering)
But I agree with the name `model` because it is a model :)